apps/modernization-ui/src/routes/AppRoutes.tsx

```diff
@@ -68,6 +69,8 @@ export const AppRoutes = () => {
                                 </Route>
                             </Route>
 
+                            <Route path="/page-builder/create-condition" element={<CreateCondition />} />
```

Comment:
this route could be moved to 
```
<Route path="/page-builder">
    ...
    <Route path="add">
        <Route path="condition" element={<CreateCondition />} />
    ...
```
to match the format of the add page route.

Reply:
Thanks Mike. Will fix it